src/user-guide/running-workflows/reflow.rst

```diff
@@ -117,19 +117,18 @@ Triggering with No Active Flows
    ``cylc trigger [--wait] ID``
 
    By default, triggered tasks will be given the flow numbers of the most
-   recent active task. This can happen, for instance, if you restart a
+   recent :term:`active tasks <active task>`. This can happen, for instance, if you restart a
    completed workflow and then trigger a task in it. The result will be the
    same as if you had triggered the task just before the workflow completed.
 
-Special Case: Triggering ``n=0`` Tasks
-   Tasks in the ``n=0`` window are :term:`active tasks <active task>`.
-   Their flow membership is already determined - that of
-   the parent tasks that spawned them.
+Special Case: Triggering ``n=0`` (:term:`active window`) Tasks
+   Tasks in the :term:`active window` already have flow membership assigned
+   (namely that of the parent tasks that spawned them).
```

Comment:
```suggestion
   Tasks in the :term:`active window` already have flow membership
   assigned. Flow membership is inherited from their parent tasks
   when they are spawned.
```

Reply:
Repeated words and phrases, in close proximity, when the subject is clear from the context kind of stand out as bad style, to me, which is distracting when reading. In this case, however, I think I'll take your suggestion with a slight tweak (I don't like the "their" there), since the second sentence sounds like an important statement in its own right.